semcore/tooltip/src/Tooltip.jsx

```diff
@@ -150,7 +150,7 @@ function TooltipPopper(props) {
   return sstyled(styles)(
     <ZIndexStackingContextProvider designToken='z-index-tooltip'>
       <Portal disablePortal={disablePortal} ignorePortalsStacking={ignorePortalsStacking}>
-        <STooltipPortalledWrapper aria-live={ariaLive} zIndex={zIndex}>
+        <STooltipPortalledWrapper role={'status'} aria-live={ariaLive} zIndex={zIndex}>
```

Comment:
I think adding role=status should be conditional, only if aria-live == polite

<img width="1343" alt="image" src="https://github.com/user-attachments/assets/03080275-5503-46bb-81c0-fa830da71cfe">

Reply:
fixed